### PR TITLE
Add highScore saved in localStorage

### PR DESCRIPTION
**Pull request by AMoldskred**
Hey, I quickly made a highScore-system for your app.

Also, my code editor automatically prettified the code:)

### CODE REVIEW

**Comment by itaditya** [comment]
Can you remove the formatting so I can see the code you have added?

File: script.js [hunk 1]
```diff
@@ -144,16 +151,35 @@ function playGame() {
 function endGame() {
   // other stuff to do when game over say saving score will be done here.
   renderGameOver();
+  handleHighScore(store.score);
+}
+
+/** Fetch or set highScore */
+function handleHighScore(newScore) {
```
Comment (AMoldskred) [inline script.js:158]: Created this function

File: script.js [hunk 2]
```diff
@@ -144,16 +151,35 @@ function playGame() {
...
+  if (newScore > currentHighScore) {
+    HS.innerHTML = Number(newScore);
+    localStorage.setItem("highScore", Number(newScore));
+    return newScore;
+  }
 }
 
 async function init() {
   const deaths = await ajaxGetDeaths();
+  store.highScore = handleHighScore();
```
Comment (AMoldskred) [inline script.js:174]: Set highScore here

File: script.js [hunk 3]
```diff
@@ -144,16 +151,35 @@ function playGame() {
 function endGame() {
   // other stuff to do when game over say saving score will be done here.
   renderGameOver();
+  handleHighScore(store.score);
```
Comment (AMoldskred) [inline script.js:154]: Updates highscore here

File: index.html [hunk 4]
```diff
@@ -11,15 +12,20 @@
   <link rel="stylesheet" href="style.css">
   <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
 </head>
+
 <body>
+  <div class="highscoreWrapper">
```
Comment (AMoldskred) [inline index.html:17]: Highscore popout here

File: style.css [hunk 5]
```diff
@@ -154,12 +156,25 @@ body {
   border: 2px solid red;
 }
 
+.highscoreWrapper {
```
Comment (AMoldskred) [inline style.css:159]: Added style here

**Comment by AMoldskred** [comment]
@itaditya I added comments on where I changed stuff. Please have a look

**Comment by itaditya** [comment]
@andreas0607 please remove the formatting. I don't want to change the formatting right now.

**Comment by itaditya** [comment]
The issue regarding this PR is closed now.

**Pull request closed without merging**